src/pages/steps/LocationStep.tsx

```diff
@@ -71,20 +76,34 @@ const API_URL = publicRuntimeConfig.apiUrl;
 
 const getGlobalValue = getValueFromTheme('global');
 
-const RecentLocations = ({ onFieldChange, ...props }) => {
+type LocationSuggestionProps = {
+  locations: Place[];
+  title: string;
+  alertVisible?: boolean;
+  isRecentLocations?: boolean;
+  onFieldChange?: (value: any) => void;
+} & StackProps;
+
+const LocationSuggestions = ({
+  onFieldChange,
+  locations,
+  title,
+  alertVisible = true,
+  isRecentLocations = true,
+  ...props
+}: LocationSuggestionProps) => {
   const { t, i18n } = useTranslation();
-  const { recentLocations: locations } = useRecentLocations();
 
   return (
-    <Stack {...props}>
+    <Stack {...props} spacing={5}>
       <Inline>
-        <Text fontWeight={'bold'}>
-          {t('create.location.recent_locations.title')}
-        </Text>
+        <Text fontWeight={'bold'}>{t(`${title}`)}</Text>
```

Comment:
I wouldn't do the translation here, it's a bit weird, since the title is coming from the parent component.
```suggestion
        <Text fontWeight={'bold'}>{title}</Text>
```